packages/workspace/src/workspace/hidden_values.ts

```diff
@@ -404,10 +404,12 @@ const filterOutHiddenChanges = async (
             changePath: path.slice(1),
           }
         }
+
         // idType === 'field'
+        const field = baseElem.fields[path[0]]
         return {
           // changeType will be undefined if the path is too short
-          changeType: baseElem.fields[path[0]].type.annotationTypes[path[1]],
+          changeType: field !== undefined ? elementAnnotationTypes(field)[path[1]] : undefined,
```

Comment:
1. we should be using `elementAnnotationTypes ` in other places where we access annotation types for an element (like line 403 for example), we don't want to have the same bug in another line...
2. it seems like the old code was able to assume the field exists, is there a reason this cannot be
```suggestion
          changeType: elementAnnotationTypes(baseElem.fields[path[0]])[path[1]],
```
?